desktop/app.js

```diff
@@ -28,6 +28,8 @@ module.exports = function main() {
   let mainWindow = null;
   let isAuthenticated;
 
+  app.on('ready', () => app.setAppUserModelId('com.automattic.simplenote'));
```

Comment:
looks great but would it benefit us to put it down at the end with the other `app.on('ready', …)` hook?

Reply:
HAHA okay

Reply:
I kept it as two function calls for symmetry's sake https://github.com/Automattic/simplenote-electron/pull/2483/commits/31ff235eb907a604ddb1d6be786eca4952918bde